shorebird_code_push/pubspec.yaml

```diff
@@ -5,9 +5,21 @@ version: 0.1.0+1
 environment:
   sdk: ">=3.0.0 <4.0.0"
 
+dependencies: 
```

Comment:
I suspect we'll need/want to reduce this set of dependencies to avoid introducing conflicts for customers.  We don't have to do that now, but likely want to leave a fixme. 🤷

Reply:
Removed, along with other vestigial Dart CLI code